test/run-tests.py

```diff
@@ -711,6 +717,8 @@ def main(args):
       parser.error('--fail-fast only works with -j1')
     if options.stop_interactive:
       parser.error('--stop-interactive only works with -j1')
+    if options.print_cmd:
```

Comment:
It just one of those commands that doesn't really make sense in the multithreaded run since that output will be interleaved.   But i guess that could be considered a personal preference?